ballerina-tests/src/test/resources/test-src/counter_test.bal

```diff
@@ -0,0 +1,47 @@
+import ballerina/observe;
```

Comment:
These were created long time back, but It's better if we can add license headers to these bal files as well.

Reply:
Added in https://github.com/ballerina-platform/module-ballerina-observe/pull/1/commits/9073340c438168a8545c0490b49c6092332b6138